### PR TITLE
Add documentation about SELinux labeling

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -129,7 +129,31 @@ Specifies the maximum number of retries XFS should attempt to complete IO when E
   Tell storage drivers to use the specified OSTree repository.  Some storage drivers, such as overlay, might use
 
 **skip_mount_home=""**
-  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
+p  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md]: rogue 'p' at start of line

File: docs/containers-storage.conf.5.md [hunk 2]
```diff
@@ -129,7 +129,31 @@ Specifies the maximum number of retries XFS should attempt to complete IO when E
...
+
+## SElinux labeling.
+
+When running on an SELinux system, if you move the containers storage graphroot directory, you must make sure the labeling is correct.
+
+Tell SELinux about the new containers storage by setting up an equivalence record.
+This tells SELinux to label content under the new path, as if it was stored
+under `/var/lib/containers/storage`.
+
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md]: I'd remove one of the blank lines here

File: docs/containers-storage.conf.5.md [hunk 3]
```diff
@@ -129,7 +129,31 @@ Specifies the maximum number of retries XFS should attempt to complete IO when E
   Tell storage drivers to use the specified OSTree repository.  Some storage drivers, such as overlay, might use
 
 **skip_mount_home=""**
-  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
+p  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
+
+## SElinux labeling.
+
+When running on an SELinux system, if you move the containers storage graphroot directory, you must make sure the labeling is correct.
+
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md]: I think this is done mostly for rootless containers?  If so, you might want to add a little follow on sentence about that.  "This is  especially useful when running a rootless container."  or some such.

File: docs/containers-storage.conf.5.md (hunk 1 above)
Comment (rhatdan) [reply docs/containers-storage.conf.5.md]: Fixed.

File: docs/containers-storage.conf.5.md (hunk 2 above)
Comment (rhatdan) [reply docs/containers-storage.conf.5.md]: Removed

File: docs/containers-storage.conf.5.md (hunk 3 above)
Comment (rhatdan) [reply docs/containers-storage.conf.5.md]: I don't agree that this done mostly for rootless containers.  The cases I see have been around people not having enough space in /var.  I will be extending this to a blog, to further my ideas on this.  But I don't think we need to mention rootless.

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**